state/protocol/badger/validity.go

```diff
@@ -68,6 +68,11 @@ func validSetup(setup *flow.EpochSetup) error {
 		return fmt.Errorf("need at least one verification node")
 	}
 
+	// first view must be before final view
+	if setup.FirstView >= setup.FinalView {
```

Comment:
it this right? shouldn't be `FirstView > FinalView`? Can they be equal?

Reply:
It might be possible to construct a one-view, one-block epoch in a test but practically these will never be equal or anywhere close to equal.